salt/modules/opkg.py

```diff
@@ -833,8 +833,8 @@ def _execute_remove_command(cmd, is_testmode, errors, jid):
             errors = [out['stderr']]
         else:
             errors = [out['stdout']]
-    else:
-        errors = []
+    elif is_testmode:
+        reportedPkgs = _parse_reported_packages_from_remove_output(out['stdout'])
```

Comment:
Why did you delete "errors = []" ?. Maybe there are callers that send None for this so they expect an array in return.